### PR TITLE
to_host utility to copy a cudf::column_view to host memory

### PR DESCRIPTION
**Pull request by devavret**
Depends on #3219

### CODE REVIEW

File: cpp/tests/utilities/column_utilities.hpp [hunk 1]
```diff
@@ -50,5 +52,39 @@ void expect_columns_equal(cudf::column_view lhs, cudf::column_view rhs);
...
+
+  if (col.size() > 0) {
+    host_data.resize(col.size());
+    CUDA_TRY(cudaMemcpy(host_data.data(), col.view().head<T>(),
+                        col.size() * sizeof(T),
+                        cudaMemcpyDeviceToHost));
+  }
+
+  if (col.nullable()) {
+    size_t num_mask_elements =
```
Comment (harrism) [inline cpp/tests/utilities/column_utilities.hpp]: I believe @jrhemstad is working on a utility function for this.

Comment (devavret) [reply cpp/tests/utilities/column_utilities.hpp]: Should I wait on that to merge this?

Comment (jrhemstad) [reply cpp/tests/utilities/column_utilities.hpp]: `num_bitmask_words` was already merged. You can update this PR to use it.

Comment (jrhemstad) [inline cpp/tests/utilities/column_utilities.hpp]:
```suggestion
    size_t num_mask_elements = num_bitmask_words(c.size());
```

Comment (devavret) [reply cpp/tests/utilities/column_utilities.hpp]: just did this

**Pull request merged**